src/Adapter/PhpredisAdapter.php

```diff
@@ -40,13 +29,16 @@ public function __construct(array $config = [], bool $lazy = true)
 
     protected function getClient(): Redis
     {
-        if (!$this->client instanceof Redis) {
-            $this->createClient($this->config);
+        if (!$this->client) {
```

Comment:
We need to apply @anesed's suggestion.
<img width="885" alt="imagen" src="https://user-images.githubusercontent.com/9433699/71826371-61448f00-3063-11ea-9cc0-ba9b5180d65e.png">

Reply:
I tried again and it is still failing.